opendcs-integration-test/build.gradle

```diff
@@ -19,39 +19,35 @@ plugins {
 }
 
 dependencies {
-    implementation(libs.testcontainers.core)
-    implementation(libs.rest.assured)
-    implementation(libs.jersey.container.servlet)
-    implementation(libs.webcompere.system.stubs.jupiter)
     implementation(libs.bundles.tomcat)
     implementation(libs.commons.io)
-    implementation(libs.jul.to.slf4j)
     implementation(libs.slf4j.api)
-    implementation(libs.opendcs)
+    implementation(libs.opendcs) {
+        exclude group: "org.glassfish.jersey.containers"
```

Comment:
We should actually fix this at the main project level. These's shouldn't be forced dependencies, they're only needed at runtime and should be provided the environment; e.g. only the install subproject of opendcs (and tests) should consume them.

Reply:
ah ok, I saw them marked as implementation in the main project

Reply:
Yeah, that's my oops. Still getting used to api vs implementation

Reply:
Moved the components to an lrgs-web jar: https://github.com/opendcs/opendcs/pull/983